src/NetTopologySuite.IO.VectorTiles.Mapbox/MapboxTileReader.cs

```diff
@@ -275,26 +301,31 @@ private static (MapboxCommandType, int) ParseCommandInteger(uint commandInteger)
         private static IAttributesTable ReadAttributeTable(Tile.Feature mbTileFeature, List<string> keys, List<Tile.Value> values)
         {
             var att = new AttributesTable();
-            for (int i = 0; i < mbTileFeature.Tags.Count; i += 2)
+
+            //Ensure there are enough keys to read.
+            if (keys.Count == mbTileFeature.Tags.Count)
```

Comment:
Don't we have to check `keys.Count  == mbTileFeature.Tags * 2` as key and value are stored intermittend?
Do we have to check this at all?

Reply:
Good call on that count comparison, I should have tested with more than one attribute. 

The reason for this test is that someone may decide they only want to read the geometries from a Mapbox tile and not the attributes. A quick and way to do this is to `tile.Layers[0].Keys.Clear();` on a Mapbox tile and the reader will not extract the attributes. That said, perhaps this is an edge case that can be done another way (clear the layers keys and each features tags). I'll remove this comparison and modify my test as needed.

Reply:
While I have your attention, where would be a good place for me to create some documentation for this library? I was going to add some to the README or the projects WIKI but wasn't sure if there was another preferred location for this.